go-controller/pkg/node/helper_linux.go

```diff
@@ -150,6 +150,6 @@ func getIntfName(gatewayIntf string) (string, error) {
 	return intfName, nil
 }
 
-func deleteConntrack(ip string, port int32, protocol kapi.Protocol) error {
-	return util.DeleteConntrack(ip, port, protocol)
+func deleteConntrack(ip string, port int32, protocol kapi.Protocol, fromOriginalTuple bool) error {
```

Comment:
can we just get rid of this function and call util.DeleteConntrack?

Reply:
yes sir! good call...